lib/license_scout/license.rb

```diff
@@ -107,7 +107,7 @@ def license_content(license_id, contents_url)
         begin
           LicenseScout::Log.debug("[license] Pulling license content for #{license_id} from #{new_url}")
           open(new_url).read
-        rescue OpenURI::HTTPError, Errno::ECONNREFUSED, SocketError
+        rescue OpenURI::HTTPError, Errno::ECONNREFUSED, SocketError, Net::OpenTimeout
```

Comment:
Is there any reason to not do this for all errors?

Reply:
Eh, probably not. I trend towards being specific but I think it's probably safe.